src/governance.cpp

```diff
@@ -954,23 +954,22 @@ void CGovernanceManager::RequestGovernanceObject(CNode* pfrom, const uint256& nH
         return;
     }
 
-    CBloomFilter filter(GOVERNANCE_FILTER_ELEMENTS, GOVERNANCE_FILTER_FP_RATE, GetRandInt(999999), BLOOM_UPDATE_ALL);
+    CBloomFilter filter;
+    filter.clear();
 
     if(fUseFilter) {
         CGovernanceObject* pObj = FindGovernanceObject(nHash);
 
         if(pObj) {
+            int nFilterElements = min(3*mnodeman.CountMasternodes(), GOVERNANCE_FILTER_MAX_ELEMENTS); 
```

Comment:
No, I mean to replace `GOVERNANCE_FILTER_MAX_ELEMENTS` i.e. to change this into
```
int nFilterElements = min(3*mnodeman.CountMasternodes(), Params.GovernanceFilterMaxElements());
```
so that testnet could send much smaller filters